generator/tree.go

```diff
@@ -145,6 +155,11 @@ func generateConfigModule(cfg *ModuleConfig, node *Node, nameToNode map[string]*
 			if !ok {
 				return // Unsupported type.
 			}
+
+			if !metricAccess(n.Access) {
+				return // Inaccessible metrics.
```

Comment:
Yes, happy to see things work well again. BTW, why will the `tableWriteOnly` \\ `tableNotify` be skipped? I'm still not clear about that.